sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -130,6 +130,7 @@ case class CatalogTable(
     createTime: Long = System.currentTimeMillis,
     lastAccessTime: Long = -1,
     properties: Map[String, String] = Map.empty,
+    catalogStats: Option[Statistics] = None,
```

Comment:
nit: according to the other names, I think we don't need the `catalog` prefix, i.e. `stats` or `statistics` is fine.